src/dlgAboutDialog.cpp

```diff
@@ -81,7 +81,7 @@ dlgAboutDialog::dlgAboutDialog(QWidget * parent) : QDialog(parent)
         }
 
         // Repeat for other text, but we know it will fit at given size
-        QString sourceCopyrightText = QChar( 169 ) % QString( " Heiko K" ) % QChar( 246 ) % QString( "hn 2008-" ) % QString(__DATE__).mid(7);
+        QString sourceCopyrightText = QChar( 169 ) % QString( " Mudlet devs 2008-" ) % QString(__DATE__).mid(7);
```

Comment:
According to that link part 4I we might want/need to include the word Copywrite i.e. ":copyright: Copywrite ..." rather than ":copyright: ..." for some jurisdictions.

On Lauchpad we use the term "[Mudlet Makers](https://launchpad.net/~mudlet-makers/+mugshots)" {humm, "Sloth" is still awaiting approval...!} - for consistency should we adopt that here as well?

Also, you have used "__Mudlet devs__" here but "__Mudlet devs.__" on the Splash screen :confused: - TBH I am not sure we should use the non-word "devs" - it strikes me as a bit slap-dash {and "Mudlet Development Team" might be a _tiny_ bit _too_ long to fit in at a reasonable size}...